server/api/server.go

```diff
@@ -36,6 +36,8 @@ func NewHandler(ctx context.Context, svr *server.Server) (http.Handler, server.S
 	router.PathPrefix(apiPrefix).Handler(negroni.New(
 		serverapi.NewRuntimeServiceValidator(svr, group),
 		serverapi.NewRedirector(svr),
+		serverapi.NewAPILimit(svr.GetConfig().PDServerCfg.APIBucketCapacity,
```

Comment:
`APIBucketCapacity` and `APIBucketRate` need to be defined.

Reply:
These have been defined in PR #4234

Reply:
got